Lib/test/test_tokenize.py

```diff
@@ -1349,6 +1349,25 @@ def assertExactTypeEqual(self, opstr, *optypes):
         self.assertEqual(token.tok_name[tokens[1 + num_optypes].exact_type],
                          token.tok_name[token.ENDMARKER])
 
+    def test_all_literal_tokens(self):
+        NON_LITERALS = {token.ENDMARKER, token.NAME, token.NUMBER,
```

Comment:
Also, I find hanging indents much more readable and they will make future diffs less noisy:

```py
non_literal_tokens = {
    token.STRING, token.NEWLINE, token.INDENT,
    ...,
    # You can add another line without having to modify the '}' so diffs will be less noisy.
    token.N_TOKENS, token.NT_OFFSET,
}
```